contracts/IdentityOracle.sol

```diff
@@ -202,37 +160,27 @@ contract IdentityOracle is DAOUpgradeableContract, ChainlinkClient, Ownable {
         }
         return result;
     }
-
-    function setJobIDIPFS1(string memory _jobid) public {
-        _onlyAvatarTmp();
-        JOBID_GET_IPFS_CID1 = _jobid;
-    }
-
-    function setJobIDIPFS2(string memory _jobid) public {
-        _onlyAvatarTmp();
-        JOBID_GET_IPFS_CID2 = _jobid;
-    }
-
-    function setJobIDStateHash(string memory _jobid) public {
+    
+    function getJobIDStateHashIPFSCID() public view returns (string memory) {
         _onlyAvatarTmp();
-        JOBID_GET_STATE_HASH = _jobid;
+        return JOBID_GET_STATE_HASH_IPFSCID;
     }
 
-    function getJobIDIPFS1() public view returns (string memory) {
+    function setJobIDStateHashIPFSCID(string memory _jobid) public {
         _onlyAvatarTmp();
```

Comment:
why is it onlyAvatarTmp?  and not  onlyAvatar?

Reply:
@sirpy Could you share with us an example or doc about how to test IdentityOracle setting dao.avatar() with the address of the deployer of the smartcontract ?

Reply:
i just ment that the name should not include "Tmp"
the avatar address should be passed in the constructor

Reply:
fixed by https://github.com/GoodDollar/ChainlinkIdentityOracle/pull/5/commits/47fdb926d0b55f857ae9ab09f895f1abdf39ef0e